boot.php

```diff
@@ -722,8 +722,6 @@ function update_db()
 		$stored = intval($build);
 		$current = intval(DB_UPDATE_VERSION);
 		if ($stored < $current) {
-			Config::load('database');
```

Comment:
The "database" config values are only loaded when there is a version difference. So we save some memory. On my system I'm having more than 1,000 "database" entries - and they aren't used when there isn't a database update.

Reply:
Yes, this is the point of this update, using memory to speed up page load. I'll come back with a modular solution where you will be able to choose your config adapter.